### PR TITLE
feat: do jobPosts component

### PR DESCRIPTION
**Pull request by SurajSulten**

### CODE REVIEW

File: client/src/modules/Job/components/titlePage/BadgeItem.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Badge, Space } from 'antd';
+
+const App: React.FC = () => {
+
+  return (
+    <Space >
+      <Badge count={44} style={{backgroundColor: "#F0F0F0", color: "#000000", opacity: 0.45,}}  />
```
Comment (antoniosk10) [inline client/src/modules/Job/components/titlePage/BadgeItem.tsx:7]: `count` should be `props`

Comment (antoniosk10) [reply client/src/modules/Job/components/titlePage/BadgeItem.tsx:7]: https://beta.reactjs.org/learn/passing-props-to-a-component

File: client/src/modules/Job/components/titlePage/BadgeItem.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Badge, Space } from 'antd';
+
+const App: React.FC = () => {
```
Comment (antoniosk10) [inline client/src/modules/Job/components/titlePage/BadgeItem.tsx:3]: your component should have the same name as your file. change `App` to `BadgeItem`

File: client/src/modules/Job/components/titlePage/TitlePage.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Space, Typography } from "antd"
+import BadgeItem from "./BadgeItem"
+
+
```
Comment (antoniosk10) [inline client/src/modules/Job/components/titlePage/TitlePage.tsx:4]: remove one space

File: client/src/modules/Job/components/titlePage/TitlePage.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+function TitlePage() {
```
Comment (antoniosk10) [inline client/src/modules/Job/components/titlePage/TitlePage.tsx:5]: try to follow by one style of code everywhere. change declaration of function as arrow function.
```suggestion
const TitlePage = ()=> {
```

**Review by antoniosk10 — APPROVED** [review]

**Review by apalchys — APPROVED** [review]

**Comment by apalchys** [comment]
@SurajSulten @antoniosk10 sorry it took so long. Please find lint/format and we will merge

**Pull request closed without merging**